src/routes/about/+page.svelte

```diff
@@ -1,41 +1,328 @@
 <svelte:head>
 	<title>About Us</title>
 	<meta name="description" content="About this app" />
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link
+		href="https://fonts.googleapis.com/css2?family=Pixelify+Sans:wght@400..700&display=swap"
+		rel="stylesheet"
+	/>
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link rel="preconnect" href="https://fonts.googleapis.com" />
+	<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+	<link
+		href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&family=Staatliches&display=swap"
+		rel="stylesheet"
+	/>
 </svelte:head>
 
-<div class="h-screen">
-	<div id="introduction">
-		<p id="about-us">About Us</p>
-		<p id="about-us-features">Features</p>
-		<p id="helpful-info">Helpful Information</p>
+<div class="content h-full bg-base inline-block">
+	<div id="grad">
+		<div id="banner-content">
+			<p class=" text-xs opacity-70 tracking-wide">Learn</p>
+			<h2 class=" mt-0 pt-0 text-5xl tracking-widest text-center font-bold p-24">
+				About<br /> Us
+			</h2>
+		</div>
 	</div>
 
-	<div id="team-information">
-		Include a section introducing the development team behind the application, highlighting key
-		members and their roles.
-		<ul>
-			<li>Team Member Name</li>
-			<li>Team Member Name</li>
-		</ul>
-	</div>
+	<div id="introduction">
+		<p class="font-semibold text-xl mb-5 ml-5">
+			Introducing <span class="typewriter-text underline decoration-accent">Living Dex Tracker</span
+			>
+		</p>
+		<p class="text-base mt-10 mb-5 ml-5 mr-5">
+			<img src="/placeholder-bulb.png" alt="bulb" class="float-right w-28" />
+			Living Dex Tracker is dolor sit amet, consectetur adipiscing elit. Proin dignissim risus eu quam
+			feugiat malesuada. Vivamus arcu sapien, feugiat sed lacus ut, malesuada congue nibh. In lacinia
+			lacus quis bibendum pulvinar. Aliquam in suscipit risus. Aliquam ut justo dolor.
+		</p>
 
-	<div id="project-details">
-		Share insights into the development process, challenges faced, and any interesting anecdotes or
-		milestones achieved during the project's journey.
-	</div>
+		<p class="font-semibold text-xl mt-20 mb-5 ml-5">Features</p>
 
-	<div id="contact-information">
-		Offer ways for users to get in touch with the development team, such as through social media, or
-		a contact form.
-	</div>
+		<!-- Ask Josh to help you with JavaScript so the card features can swipe -->
```

Comment:
Replace this comment with a TODO comment, e.g. `TODO (#41) - Make this a card carousel`